srcpkgs/linux5.13-surface/template

```diff
@@ -0,0 +1,303 @@
+# Template file for 'linux5.13-surface'
+pkgname=linux5.13-surface
+version=5.13.13
+revision=3
+wrksrc="linux-${version%.*}"
+short_desc="Linux kernel and modules containing patches and drivers for Microsoft Surface series devices ( series)"
+maintainer="Tino Heuberger <tinoheuberger@protonmail.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/linux-surface/linux-surface"
+distfiles="https://cdn.kernel.org/pub/linux/kernel/v5.x/linux-${version%.*}.tar.xz
+ https://cdn.kernel.org/pub/linux/kernel/v5.x/patch-${version}.xz 
+ https://github.com/linux-surface/linux-surface/archive/refs/tags/arch-${version}-${revision}.tar.gz>surface-linux.tar.gz"
+checksum="3f6baa97f37518439f51df2e4f3d65a822ca5ff016aa8e60d2cc53b95a6c89d9
+ db626e41d43e74f5c26901dc71e19c6792eed902f44abf0a509ca4a9a2617cb8
+ a0e62b0268950919d58199da878c3536b35da76745053947e5480a66568bb35f"
+skip_extraction="patch-${version}.xz"
+python_version=3
+create_wrksrc=yes
+
+# Only x86_64 is valid for Microsoft Surface devices
+archs="x86_64*"
+
+nodebug=yes  # -dbg package is generated below manually
+nostrip=yes
+noverifyrdeps=yes
+noshlibprovides=yes
+preserve=yes
+
+hostmakedepends="tar xz bc elfutils-devel flex gmp-devel kmod libmpc-devel
+ openssl-devel perl uboot-mkimage cpio pahole python3"
+
+_kernver="${version}_${revision}"
+triggers="kernel-hooks"
+kernel_hooks_version="${_kernver}"
+versionSplitted=( ${version//./ } )
+majorVersion="${versionSplitted[0]}.${versionSplitted[1]}"
+
+# These files could be modified when an external module is built.
+mutable_files="
+ /usr/lib/modules/${_kernver}/modules.builtin.bin
+ /usr/lib/modules/${_kernver}/modules.builtin.alias.bin
+ /usr/lib/modules/${_kernver}/modules.softdep
+ /usr/lib/modules/${_kernver}/modules.dep
+ /usr/lib/modules/${_kernver}/modules.dep.bin
+ /usr/lib/modules/${_kernver}/modules.symbols
+ /usr/lib/modules/${_kernver}/modules.symbols.bin
+ /usr/lib/modules/${_kernver}/modules.alias
+ /usr/lib/modules/${_kernver}/modules.alias.bin
+ /usr/lib/modules/${_kernver}/modules.devname"
+
+# reproducible build
+export KBUILD_BUILD_TIMESTAMP=$(LC_ALL=C date -ud @${SOURCE_DATE_EPOCH:-0})
+export KBUILD_BUILD_USER=voidlinux
+export KBUILD_BUILD_HOST=voidlinux
+
+if [ "$CROSS_BUILD" ]; then
+	_cross="CROSS_COMPILE=${XBPS_CROSS_TRIPLET}-"
+fi
+
+pre_patch() {
+    cd $XBPS_BUILDDIR/linux-$majorVersion/linux-$majorVersion
+	xzcat $XBPS_SRCDISTDIR/$pkgname-$version/patch-${version}.xz | patch -Np1
+
+    # Apply Surface linux patches
+    for i in $XBPS_BUILDDIR/linux-$majorVersion/linux-surface-arch-${version}-${revision}/patches/${majorVersion}/*.patch; do patch -Np1 < $i; done
+}
+
+do_configure() {
+	# If there's a file called <arch>-dotconfig, use it to
+	# configure the kernel; otherwise use arch defaults and all stuff
+	# as modules (allmodconfig).
+	local arch subarch
+    cd $XBPS_BUILDDIR/linux-$majorVersion/linux-$majorVersion
```

Comment:
Set build_wrksrc, and look at manual what is current working directory for functions, most of cd won't be needed.
Fix indentaton.

Reply:
Done